null_safety_examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Flutter null safety example apps
+
+To analyze, test and run individual apps, execute the following commands from
+the repo root (`$PROJECT` represents the app project path, such as
+`null_safety_examples/layout/base`):
+
+1. `flutter create --no-overwrite $PROJECT`
+1. `cd $PROJECT`
+1. `flutter analyze`
+1. `flutter test`
+1. `flutter run`
+
+To learn more about setting up Flutter and running apps, see
+[flutter.dev/get-started][].
+
+[flutter.dev/get-started]: https://flutter.dev/docs/get-started
```

Comment:
A relative link should be sufficient and also works better if you've forked the repo:   `/docs/get-started`.

Reply:
Done.